lib/rex/proto/ms_dtyp.rb

```diff
@@ -23,6 +23,22 @@ class MsDtypAccessMask < BinData::Record
     bit2   :reserved1
     bit1   :ma
     bit1   :as
+    def bit_names
+      names = []
+      names << :GENERIC_READ if self.gr
+      names << :GENERIC_WRITE if self.gw
+      names << :GENERIC_EXECUTE if self.gx
+      names << :GENERIC_ALL if self.ga
+      names << :MAXIMUM_ALLOWED if self.ma
+      names << :ACCESS_SYSTEM_SECURITY if self.as
+      names << :SYNCHRONIZE if self.sy
+      names << :WRITE_OWNER if self.wo
+      names << :WRITE_DACL if self.wd
+      names << :READ_CONTROL if self.rc
+      names << :DELETE if self.de
```

Comment:
All of these are going to show up every time because 0 evaluates to true in Ruby.  These each need to be `... if self... != 0`

Reply:
Changed accordingly